serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -19,19 +19,14 @@
 ** xref:core/working-with-parallelism.adoc[Parallelism]
 // We will refactor this section here: https://issues.redhat.com/browse/KOGITO-9451
 * Tooling
-** xref:tooling/serverless-workflow-editor/swf-editor-overview.adoc[Serverless Workflow editor]
-** xref:tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc[VS Code extension for Serverless Workflow editor]
-** xref:tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc[Chrome GitHub extension for Serverless Workflow editor]
-** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-overview.adoc[Kogito Serverless Workflow Tools extension in Quarkus Dev UI]
-*** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-instances-page.adoc[Workflow Instances in Kogito Serverless Workflow Tools extension]
-*** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc[Workflow Definitions in Kogito Serverless Workflow Tools extension]
-*** xref:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-custom-dashboard-page.adoc[Dashboards in Kogito Serverless Workflow Tools extension]
-** xref:tooling/kn-plugin-workflow-overview.adoc[{product_name} plug-in for Knative CLI]
+** xref:tooling/serverless-workflow-editor/swf-editor-overview.adoc[Editor]
+** xref:tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc[VS Code extension]
 ** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-overview.adoc[{serverless_logic_web_tools_name}]
-*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-github-integration.adoc[Integrating your workflow project in GitHub using {serverless_logic_web_tools_name}]
-*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc[Integrating your workflow project with OpenShift using {serverless_logic_web_tools_name}]
-*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-redhat-application-services-integration.adoc[Integrating with Red Hat OpenShift Application and Data Services]
-*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc[Deploying your workflow projects using {serverless_logic_web_tools_name}]
+*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-github-integration.adoc[Integration with GitHub]
+*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc[Integration with OpenShift]
+*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-redhat-application-services-integration.adoc[Integration with Service Registries]
+*** xref:tooling/serverless-logic-web-tools/serverless-logic-web-tools-deploy-projects.adoc[Deployment]
+** xref:tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc[Chrome extension for GitHub]
```

Comment:
Maybe better to put it just after the VS Code extension?

Reply:
Done.